exporter/trace/trace_proto.go

```diff
@@ -69,10 +69,14 @@ func protoFromSpanData(s *export.SpanData, projectID string) *tracepb.Span {
 	spanIDString := s.SpanContext.SpanID.String()
 
 	name := s.Name
```

Comment:
Please rename this variable to `displayName` - Span has the `Name` field which will have completely different value (see below).
Also, the initialization with `s.Name` value seems redundant since it will always be overwritten below.

Reply:
I would actually suggest to extract this entire code block into `generateDisplayName(s, format)` to make the code cleaner / easier to read.